OxfordBodleian/Juel-Jensen/BDLaethg45.xml

```diff
@@ -195,7 +195,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <handNote xml:id="h1" script="Ethiopic">
                                 <date notBefore="1800" notAfter="2006">Nineteenth–twenty-first century.</date>
                                 <seg type="ink">Black, red.</seg>
-                                <desc>Mediocre but regular hand, cursive script.</desc>                            
+                                <desc>Mediocre but regular hand, cursive script.</desc>
```

Comment:
(Not part of Eliana's revision, but @thea-m and @MassimoVilla , is "cursive" a good word here?)